### ISSUE TITLE
Community License RuntimeError

### ISSUE THREAD

**Issue by RSchellenberger** [ampl/ampls-api#6]
Hello. With some models I get a Runtime Error when attempting to .exportGurobiModel() an amplpy.AMPL object into the ampls-api for use with a community licensed environment. However, when I run the same example in a fully licensed environment it works fine. 

It is also worth mentioning that it is not all models that have this behavior, running the same examples will always error in the community licensed environment but running some other models will always work just fine.

Additionally, if I stay within amplpy and use amplpy.AMPL.solve() without exporting to gurobi model it works just fine...

Not sure what's going on here. I have attached a python script that will always error in my community licensed environment and will never error in my full licensed environment.

[recreateAmplpyGurobiRuntimeError.py.txt](https://github.com/ampl/ampls-api/files/11421667/recreateAmplpyGurobiRuntimeError.py.txt)

**Comment by RSchellenberger** [comment]
Let me know if this issue should go in the amplpy repo (https://github.com/ampl/amplpy) instead and I can close this issue and copy it over there.

**Comment by fdabrandao** [comment]
Thanks for reporting this issue and for providing a reproducible example. We have just released amplpy-gurobi/ampl-cplex/amplpy-xpress v0.1 and this issue has been addressed (we changed the license validation procedure):
```bash
$ python -m pip install amplpy_gurobi==0.1 --upgrade
$ python recreateAmplpyGurobiRuntimeError.py
Gurobi 10.0.1: optimal solution; objective 25
2 simplex iterations
Objective Value = 25.0
amplpy solve with 10 variables complete
Gurobi 10.0.1: optimal solution; objective 2550
0 simplex iterations
Objective Value = 2550.0
amplpy solve with 101 variables complete
Gurobi 10.0.1: optimal solution; objective 25
2 simplex iterations

Gurobi 10.0.1: optimal solution; objective 25
2 simplex iterations
Objective Value = 25.0
amplpy_gurobi solve with 10 variables complete
Gurobi 10.0.1: optimal solution; objective 2550
0 simplex iterations

Gurobi 10.0.1: optimal solution; objective 2550
0 simplex iterations
Objective Value = 2550.0
amplpy_gurobi solve with 101 variables complete
```
Please let us know if the problem persists or if any new issue arrises. This was a major update but we tried to keep backward compatibility with the previous version.
